### DISCUSSION TITLE
Invitation to Slack

### DISCUSSION THREAD

**Discussion by abumalick** [solidusio/solidus#5348] [Troubleshooting] [answered]
Hey there!

I would like to join solidus community in slack as I am new to solidus and would benefit from experience of the community.

Can you send me an invite?

***@gmail.com

**Comment by jarednorman** [comment, marked as answer]
I've sent you an invite. I'll look into what's going on, why you can't request invites anymore.

**Reply by jarednorman** [reply]
@aldesantis is going to look into this tomorrow.
